docs/DeveloperGuide.md

```diff
@@ -204,6 +199,16 @@ The following activity diagram summarizes what happens when a QR Code is generat
 
 _vCard_ is a data format for contact information. Detailed information can be found in [RFC 6350](https://datatracker.ietf.org/doc/html/rfc6350).
 
+### Adding a Note To An Existing Client
+
+The sequence diagram below shows how the components interact with each other when the user inputs the command `note 1 note/Likes to eat`.
+
+![AddNoteSequenceDiagram](images/AddNoteSequenceDiagram.png)
+
+The diagram highlights the four main components of FitBook, highlighted in their respective colors. For more information regarding the four main components, see [Main components of the architecture](#main-components-of-the-architecture).
```

Comment:
The hyperlink here does not seem to be working. 

I believe that markdown can only natively link to headers and not bolded text. Maybe use a HTML anchor tag instead?

Reply:
Good catch! Will fix.